apps/masterbots.ai/lib/hooks/use-thread.tsx

```
@@ -206,110 +205,103 @@ export function ThreadProvider({ children }: ThreadProviderProps) {
+        typeof value === "function" ? value(prev.isNewResponse) : value,
+    }));
+
+  const setIsOpenPopup: React.Dispatch<React.SetStateAction<boolean>> = (
+    isOpen,
+  ) =>
+    setState((prev) => ({
+      isOpenPopup:
+        typeof isOpen === "function" ? isOpen(prev.isOpenPopup) : isOpen,
+    }));
 
   const value = React.useMemo(
```

Biome

[error] This hook does not specify all of its dependencies: getRandomChatbot

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)